java/src/main/java/ai/rapids/cudf/ColumnFilterOptions.java

```diff
@@ -62,5 +71,10 @@ public T includeColumn(Collection<String> names) {
       includeColumnNames.addAll(names);
       return (T) this;
     }
+
+    public T decimal128Column(String... names) {
```

Comment:
nit: Could we get some javadocs on this API? It is not clear exactly how to use this API.

Reply:
Actually now that I think about it. These right now are ORC specific.  I would rather have this just be a part of the ORCOptions class.

Reply:
Done.